tox.ini

```diff
@@ -1,6 +1,6 @@
 [tox]
 minversion = 2.4.1
-envlist = py27,pypy,py33,py34,py35,pep8,coverage,docs
+envlist = py27,pypy,py33,py34,py35,py36,pep8,coverage,docs
```

Comment:
I think we can remove py33 here also. And also Python 3.3 from the classifiers in setup.py.

Reply:
I've updated the PR with your suggestions